build.rs

```diff
@@ -0,0 +1,67 @@
+use std::{collections::BTreeMap, fs};
+
+use anyhow::Result;
+use rgbstd::{
+    interface::{LIB_ID_RGB20, LIB_ID_RGB21, LIB_ID_RGB25},
+    stl::{LIB_ID_RGB, LIB_ID_RGB_CONTRACT, LIB_ID_RGB_STD},
```

Comment:
`LIB_ID_RGB_STD` is not about consensus: if it changes it means you may need to update consignments or indexes or serialization for stash - but not the fact that you need to re-issue and migrate the assets

`LIB_ID_RGB20, LIB_ID_RGB21, LIB_ID_RGB25` - this will mean you will need to do another interface implementations (which is trivial) and doesn't need to re-issue or migrate the assets

`LIB_ID_RGB` - this is the only consensus thing which (if changes) means assets have to be re-issued

Reply:
Thanks to clarify @dr-orlovsky